buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidAppTarget.groovy

```diff
@@ -100,8 +100,8 @@ class AndroidAppTarget extends AndroidLibTarget {
             String last // can denote either module or configuration name
             boolean fullyQualified = false
 
-            if (exoPackageDep.contains(";")) {
```

Comment:
';' is correct. This is how we separate dependencies which are specified like group:module.

Please change it back